x/ibc/04-channel/handler.go

```diff
@@ -27,7 +25,7 @@ func HandleMsgChannelOpenInit(ctx sdk.Context, k keeper.Keeper, portCap *capabil
 			sdk.NewAttribute(types.AttributeKeyChannelID, msg.ChannelID),
 			sdk.NewAttribute(types.AttributeCounterpartyPortID, msg.Channel.Counterparty.PortID),
 			sdk.NewAttribute(types.AttributeCounterpartyChannelID, msg.Channel.Counterparty.ChannelID),
-			sdk.NewAttribute(types.AttributeKeyConnectionID, msg.Channel.ConnectionHops[0]), // TODO: iterate
```

Comment:
The attribute key is `connectionID` the TODO didn't feel helpful (or necessary?)

Reply:
this was a TODO for when multiple hops are allowed

Reply:
is there an issue we can add it to? I think a different attribute key would be added for connection hops since this is for `connectionID`